test/gpu/native/environment/gasnet.compopts

```diff
@@ -0,0 +1 @@
+--comm gasnet
```

Comment:
Just FYI, we have `.compenv` files. So, what you actually need here is `gasnet.compenv` file that has `CHPL_COMM=gasnet` in it.

Or that and this.. But I'm fine as-is

Reply:
If we really wanted to be exhaustive about testing we should probably do both. I think I'll leave it as is. But thanks for letting me know about `.compenv` (I wasn't aware of it).